js/interface-lists.js

```diff
@@ -452,8 +481,7 @@ function attahObservers() {
             }, 4000);
             return;
           } else {
-            $('.has-error').removeClass('has-error');
-            $('.component-error').addClass('hidden');
+            showOrHideError(false);
```

Comment:
I notice a `field` isn't provided here, but there's nothing in the function that deals with what happens if the second parameter isn't provided.

Reply:
We deal with this by if clause which fires when we provide a `true` as the first parameter and exit from the function by return in if clause. In another situation, we remove error classes by referring to them by classes.